package.json

```diff
@@ -116,26 +116,27 @@
   },
   "resolutions": {
     "@types/react": "*",
+    "browserslist": "^4.16.5",
     "clean-css": "^4.2.3",
+    "dns-packet": "^1.3.4",
+    "favicons/colors": "1.4.0",
     "glob-parent": "^5.1.2",
     "immer": "^9.0.6",
     "is-svg": "^4.2.2",
     "jimp": "^0.16.1",
     "jpeg-js": "^0.4.3",
     "js-yaml": "^3.13.1",
+    "karma/colors": "1.4.0",
+    "lodash": "^4.17.21",
+    "nanoid": "^3.1.31",
     "node-notifier": "^9.0.0",
-    "trim": "^0.0.3",
     "trim-newlines": "^4.0.2",
+    "trim": "^0.0.3",
```

Comment:
nit: as this probably should be in the original location if alphabetizing the dependencies.